pushy/src/main/java/com/relayrides/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -63,8 +65,10 @@
     private static final String SOUND_KEY = "sound";
     private static final String CATEGORY_KEY = "category";
     private static final String CONTENT_AVAILABLE_KEY = "content-available";
+    private static final String MUTABLE_CONTENT_KEY = "mutable-content";
 
     private static final String ALERT_TITLE_KEY = "title";
+    private static final String ALERT_SUBTITLE_KEY = "subtitle";
```

Comment:
Testing reveals that these *do* exist. I'm happy to merge this and add them on my own, though.